### DISCUSSION TITLE
simple divider between collections

### DISCUSSION THREAD

**Discussion by Lukas-Sachse** [directus/directus#10426] [Feature Requests]
In the image on Producthunt are these dividers between collections:

![Group 1](https://user-images.githubusercontent.com/62059415/145466278-e4e09dff-c476-47c2-90a3-740b6ef09e9f.png)

According to @benhaynes on Discord these are the _old_ dividers before the _new_ folder approach has replaced them.
Im interested in getting these kind of dividers back.

**Comment by benhaynes** [comment]
Would love to get these in... the question is: How are these configured... and how are they saved?

We're saving everything as Collections (or Collection Aliases for folders) now, so that won't work. @rijkvanzanten any ideas?

**Reply by benhaynes** [reply]
Also, it'd be nice to have the future ability to support other things, like: custom links, etc.

**Reply by snorbik** [reply]
@benhaynes Any update regarding these dividers? We would appreciate them as well.

**Reply by benhaynes** [reply]
Nope, we're currently focused on stability/docs/tests... not features.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
